app/uk/gov/hmrc/taxenrolmentassignmentfrontend/views/UnderConstructionView.scala.html

```diff
@@ -0,0 +1,24 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(layout: templates.Layout)
+
+@()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@layout(pageTitle = Some(messages("multiplePTE.title"))) {
+<h1 class="govuk-heading-xl">@{messages("multiplePTE.heading")}</h1>
+<p class="govuk-body">@{messages("multiplePTE.text")}</p>
```

Comment:
could pass in a message for the body to state type of multiple account. i.e another account with PT enrolment, no accounts with PT enrolment etc